### PR TITLE
Event Hub: Update SDK for Swagger changes

### PR DESCRIPTION
**Pull request by v-Ajnava**
## Description 
Event Hub - 
Update SDK for : 
Swagger property changes reverted (SDK was not updated) and fixes
Namespace patch operation added 
Version update

Swagger PR Links : 
https://github.com/Azure/azure-rest-api-specs/pull/1111
https://github.com/Azure/azure-rest-api-specs/pull/1132

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [X] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by v-Ajnava** [comment]
will fix the errors and update the PR.

File: src/SDKs/EventHub/EventHub.Tests/EventHub.Tests.csproj [hunk 1]
```diff
@@ -1,29 +1,50 @@
...
-    <AssemblyName>EventHub.Tests</AssemblyName>    
-  </PropertyGroup>
-  <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <AssemblyName>EventHub.Tests</AssemblyName>
+    <AssemblyOriginatorKeyFile>../keyfile.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PackageId>EventHub.Tests</PackageId>
```
Comment (shahabhijeet) [inline src/SDKs/EventHub/EventHub.Tests/EventHub.Tests.csproj]: @v-Ajnava please let me know if you were having any difficulty in building the project, that prompted you for this change.
Please reach out to me and I can explain if things were either not building or understand what you wanted to do.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Review by shahabhijeet — APPROVED** [review]

File: src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj [hunk 2]
```diff
@@ -9,5 +9,6 @@
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <Version>1.2.0</Version>
```
Comment (shahabhijeet) [inline src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj]: @v-Ajnava no need to have <version> <versionPrefix> at the same time

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj (hunk 2 above)
Comment (v-Ajnava) [inline src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj]: Used VersionPrefix

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**